src/wiki/help-pages/Java-settings.md

```diff
@@ -8,18 +8,18 @@ eleventyNavigation:
 
 In this page you can set the global Java settings.
 
-![Java tab under Prism Launcher settings](/img/Java-settings.png)
+{% image "Java tab under Prism Launcher settings", "./src/img/screenshots/JavaSettingsLight.png", "./src/img/screenshots/JavaSettingsDark.png" %}
 
 ## Memory
 
-![Memory section of java tab under Prism Launcher settings](/img/Java-settings-memory.png)
+{% image "Memory section of java tab under Prism Launcher settings", "./src/img/screenshots/JavaSettingsMemoryLight.png", "./src/img/screenshots/JavaSettingsMemoryDark.png" %}
```

Comment:
use [](), what you did is breaking stuff on mobile

Reply:
I don't think `[]()` has support for light and dark theme, fixed it in another way so that it also works on mobile